src/coreclr/jit/abi.cpp

```diff
@@ -154,6 +154,29 @@ var_types ABIPassingSegment::GetRegisterType() const
     }
 }
 
+//-----------------------------------------------------------------------------
+// GetRegisterType:
+//  Return the smallest type larger or equal to Size that most naturally
+//  represents the register this segment is passed in, taking into account the
+//  GC info of the specified layout.
```

Comment:
nit: the formatting is a bit off (2 spaces vs 3 spaces for Return value, also, missing arg section)

Reply:
Let me address that in a follow-up